app/views/ajax/home.html.erb

```diff
@@ -50,5 +138,90 @@ document.addEventListener('DOMContentLoaded', (event) => {
     console.log("sending data:",dataToSend)
     eventRequest.send(dataToSend);
   });
+
+
+  const addItemButton = document.getElementById('add-item');
+  const item_name = document.getElementById('item-name');
+  const item_todo_id = document.getElementById('item-todo-id')
+  //const checkbox = document.getElementById("done_check").checked;
+  const checkbox = document.querySelector('input[value="Done"]');
+  addItemButton.addEventListener('click', (event) => {
+    var eventRequest = new XMLHttpRequest();
+    eventRequest.onreadystatechange = function () {
+      if(eventRequest.readyState === XMLHttpRequest.DONE) {
+        if  (eventRequest.status === 201) {     // this is the expected return code for a POST to create a todo
+          //resultsDiv.textContent=eventRequest.responseText; // this will be the todo that is created
+          alert("Item Successfully Added to ToDo list")
+        } else {
+          alert(`Return code ${eventRequest.status} ${eventRequest.statusText} ${eventRequest.responseText}`);  // error handling
+        }
+      }
+    };
+    eventRequest.open('POST', 'http://localhost:3001/todos/'+ item_todo_id.value +'/items');  // method and URL to create a todo
+    eventRequest.setRequestHeader("Content-Type", "application/json");  // always do this on POST or PUT
+    let dataToSend = `{"name": "${item_name.value}", "done": ${checkbox.checked}}` // JSON for the create
```

Comment:
There is actually a nicer way to do this, which my example should have used.  You could have:
var dataToSend2 = {name: itemName, done: doneValue};
followed by
eventRequest.send(JSON.stringify(dataToSend2));
That way the stringify does the serialization for you.